mxtoolbox_dns/help.md

```diff
@@ -1,11 +1,17 @@
 # Description
 
-[mxtoolbox_dns](https://www.mxtoolbox.com) is a Interface with MxToolBox DNS lookup API
```

Comment:
We should be keeping the links

Reply:
It remains, it was just moved from the start of the paragraph to the end.